ViAn/GUI/reportgenerator.cpp

```diff
@@ -170,12 +167,12 @@ std::string ReportGenerator::date_time_generator() {
  * The name will be based on the project and current date and time.
  * @param activeDocument, the document that is selected
  */
```

Comment:
I'd rather not change this since it will be changed in base branch as well. Seems unneccesary with two way confligcts.